### ISSUE TITLE
Schutz gegen Manipulation bei Rückspielung von Datensicherung

### ISSUE THREAD

**Issue by littflo** [a-sit-plus/at-registrierkassen-mustercode#464]
Hallo, ich habe eine Frage zum Manipulationsschutz beim Rückspielen einer Datensicherung.
Angenommen der Benutzer unserer Kassensoftware macht heute eine Datensicherung (inkl. 
DEP). Nach 2 Tagen tritt ein Defekt der Festplatte auf. Das System muss neu installiert
werden und die Sicherung von vor 2 Tagen wird eingespielt. Alle Belege, die in den
letzten 2 Tagen generiert wurden sind verloren. Der Umsatzzähler ist auf dem Stand von
vor 2 Tagen. 
Wenn nun ein neuer Beleg erstellt wird, wird eine Belegnummer generiert, die schon verwendet wurde.

Wie wird nun sichergestellt, dass die Steuer für die verloreren Beleg abgeführt wird?

Bei einer Finanzprüfung sieht man zwar, dass es eine Lücke von 2 Tagen gibt, aber es kann
ja sein, dass der Betrieb mal 2 Tage geschlossen hatte. Solange nicht ein Beleg eines
Kunden aus dem verlorenen Zeitraum auftaucht, ist die Manipulation doch nicht 
nachweisbar, oder doch?

Gibt es irgendwelche Vorkehrungen, die ich als Kassensoftwarehersteller beim Rückspielen
einer Datensicherung treffen kann, um Manipulation zu verhindern?

**Comment by ErichFreitag** [comment]
Sie können nach RKSV keine Datensicherung rückspielen, weil sonst die Belegverkettung verloren geht.

Die Kasse ist ab dem Defekt ebenso defekt und außer Betrieb zu nehmen. Bis eine neue Kasse vorhanden ist müssen Belege händisch erfasst und danach nacherfasst werden. Die neue Kasse ist neu in Betrieb zu nehmen.

**Comment by littflo** [comment]
Danke für die schnelle Antwort.
Aber in dem genannten Szenario wäre doch die Belegverkettung aus technischer Sicht fehlerfrei, da ja alle Belege ab der Datensicherung gelöscht sind und neue Belege erstellt werden, die dann ja korrekt verkettet werden.
Die Verkettung verhindert Manipulation bei Belegen in der Mitte der Kette. Aber ein Abschneiden eines Bereichs am Ende der Kette wird nicht verhindert und ermöglicht damit Manipulation. Oder übersehe ich da etwas?

Das Problem das ich sehe ist, dass ich nicht verhindern kann, dass sich ein Benutzer eine Sicherung der Datenbank anfertigt und diese nach ein paar Tagen wieder einspielt.

**Comment by ErichFreitag** [comment]
Da kann ich ihnen nicht helfen - das Procedere ist klar. Und Belegnummern müssen außerdem auf Lebenszeit einer Kasse eindeutig sein.

Verhindern können sie es nicht. Sie können nur ihren Kunden darauf hinweisen, wie er bei einem Defekt vorzugehen hat. Dann ist es nicht mehr ihre Verantwortung, was jemand tatsächlich tut.

**Comment by littflo** [comment]
Ja, das wird wohl die einzige Möglichkeit sein. Danke für Ihre Hilfe!
